presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -81,6 +81,7 @@
 
     private final TestingDiscoveryServer discoveryServer;
     private final TestingPrestoServer coordinator;
+    private TestingPrestoServer backupCoordinator;
```

Comment:
final

Reply:
The backupCoordinator is only used to test system connector, so I change it to local variabale.

Reply:
Is it ok? @losipiuk

Reply:
I would prefer to keep it as a field and add getter for it to keep it consistent with `coordinator` (even though we do not need to access it externally so far).

Reply:
Thanks, backup coordinator based on existence of backupCoordinatorProperties, so I keep it as a field but use Optional, and I add getter for it.

Reply:
> I would prefer to keep it as a field and add getter for it to keep it consistent with `coordinator` (even though we do not need to access it externally so far).

Thanks, backup coordinator based on existence of backupCoordinatorProperties, so I keep it as a field but use Optional, and I add getter for it.

Reply:
@losipiuk Is it ok for this change?